src/hooks/useAutoDismissMobileKyeboardHandler.ts

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useRef } from 'react';
+
+import { isIOSMobile } from '../utils';
+
+const INPUT_ELEMENT_SELECTOR = '.sendbird-message-input';
+const SEND_BUTTON_SELECTOR = '.sendbird-message-input--send';
+
+function useAutoDismissMobileKeyboardHandler(): void {
+  const addedButtons = useRef<HTMLElement[]>([]);
+
+  useEffect(() => {
+    const handleDismissKeyboard = (): void => {
+      setTimeout(() => {
+        if (document.activeElement instanceof HTMLElement) {
+          // blur the active element(send button) to dismiss the keyboard on mobile
+          document.activeElement.blur();
+        }
+      }, 200);
+    };
+
+    const handleKeyDown = (event: KeyboardEvent): void => {
+      if (event.key === 'Enter' && isIOSMobile) {
+        handleDismissKeyboard();
+      }
+    };
+
+    const observerCallback = (mutations: MutationRecord[]): void => {
+      mutations.forEach((mutation) => {
+        if (mutation.type === 'childList') {
+          mutation.addedNodes.forEach((node) => {
```

Comment:
```suggestion
        if (mutation.type === 'childList') {
          // here
          mutation.addedNodes.forEach((node) => {
```

저 위치에서도 한번 클린업이 필요할 것 같아 보입니닷

```js
addedButtons.current.forEach((button) =>
  button.removeEventListener('click', handleDismissKeyboard)
);
```

Reply:
아 아니면 addEventListener 직전에 removeEventListener 를 하는편이 더 나을수도 있겠네요

Reply:
https://github.com/sendbird/chat-ai-widget/pull/129/commits/bda037feb29c829a05b43ed1028a4c1fd988e828 넵 그게 좋겠네요 여기 반영했습니다!